run_worker.py

```diff
@@ -0,0 +1,24 @@
+# @@@SNIPSTART python-money-transfer-project-template-run-worker
+import asyncio
+
+from temporalio.client import Client
+from temporalio.worker import Worker
+
+from shared import MONEY_TRANSFER_TASK_QUEUE_NAME
+from activities import withdraw, deposit, refund
+from workflows import MoneyTransfer
+
+async def main() -> None:
+    client: Client = await Client.connect("localhost:7233", namespace="default")
+    # Run the worker
+    worker: Worker = Worker(
+        client, task_queue=MONEY_TRANSFER_TASK_QUEUE_NAME,
+                workflows=[MoneyTransfer],
+                activities=[withdraw, deposit, refund]
```

Comment:
The indention is weird here. Would encouraging using "black" or similar formatter.